Source/WebCore/page/Quirks.h

```diff
@@ -219,6 +219,7 @@ class Quirks {
 
     bool shouldDisableElementFullscreenQuirk() const;
     bool shouldIgnorePlaysInlineRequirementQuirk() const;
+    bool shouldInjectCSSInFullscreen() const;
```

Comment:
This name is probably too generic, there are probably going to be other quirks that inject CSS in fullscreen.